gen/gen.go

```diff
@@ -309,6 +309,6 @@ func (s *s) ReadDoc() string {
 }
 
 func init() {
-	swag.Register(swag.Name, &s{})
+	swag.Register({{ printf "%q + %q" .Host .BasePath}}, &s{})
```

Comment:
What is the expected behavior In the event of many API applications don't have a Host and Basepath defined?